ReactAndroid/src/main/java/com/facebook/react/bridge/queue/MessageQueueHandlerThread.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react.bridge.queue;
+
+import android.os.Looper;
+
+class MessageQueueHandlerThread extends Thread {
+  private Looper mLooper;
+
+  protected MessageQueueHandlerThread(String name, long stackSize) {
+    super(null, null, name, stackSize);
+  }
+
+  private void onLooperPrepared() {
+  }
+
+  @Override
+  public void run() {
+    Looper.prepare();
+    synchronized (this) {
+      mLooper = Looper.myLooper();
+      notifyAll();
+    }
+    onLooperPrepared();
```

Comment:
I have removed unused onLooperPrepared method,added Process.setThreadPriority(Process.THREAD_PRIORITY_DISPLAY) code into MessageQueueHandlerThread and updated the code.